assets/scripts/auth/api-user.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const app = require('../app-data');
+
+const signUp = (success, failure, data) => {
+  $.ajax({
+    method: "POST",
+    url: app.api + '/sign-up',
+    data,
+
+  })
+  .done(success)
```

Comment:
A few things:

- success and failure shouldn't be something that is passed to signIn() in the first place.
- line 16: change to const signIn = (data) => {
- line 23: change to }).done((success) => console.log(success))
- line 24: change to .fail((fail) => console.error(fail));

Basically, change the things in .done and .fail to callbacks, kind of like with promises.